esp/services/ws_workunits/ws_workunitsHelpers.hpp

```diff
@@ -139,6 +141,8 @@ class WsWuInfo
     IEspWUArchiveModule* readArchiveModuleAttr(IPropertyTree& moduleTree, const char* path);
     void readArchiveFiles(IPropertyTree* archiveTree, const char* path, IArrayOf<IEspWUArchiveFile>& files);
     void outputALine(size32_t len, const char* content, MemoryBuffer& outputBuf, IFileIOStream* outIOS);
+    unsigned readLogProcessID(const char* linePtr);
+    void readWorkunitLog(IBufferedFileIOStream* ios, MemoryBuffer& buf, const char* outFile);
```

Comment:
see other comment, not sure 'MemoryBuffer& buf' is used, should be removed if not.